python/simpler/worker.py

```
@@ -3792,7 +4060,17 @@ def _close_l3_l2_orch_comm(self) -> None:
                 pass
         self._live_l3_l2_regions.clear()
+        self._l3_l2_orch_comm_clients.clear()
+        self._host_request_admission_clients.clear()
+        self._l3_l2_orch_comm_ready.clear()
         self._l3_l2_orch_comm_host_buffers.clear()
+        for shm in self._l3_l2_orch_comm_shms.values():
+            try:
+                shm.close()
+                shm.unlink()
+            except Exception:  # noqa: BLE001
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)